### PR TITLE
lanaguage, communicates, communication between client and server

### PR DESCRIPTION
**Pull request by rivalez**

### CODE REVIEW

File: battleship-server/src/main/java/com/epam/solid/nie/config/language/CommunicateProviderImpl.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package com.epam.solid.nie.config.language;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ResourceBundle;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+public class CommunicateProviderImpl implements CommunicateProvider {
```
Comment (Ozymadias) [inline battleship-server/src/main/java/com/epam/solid/nie/config/language/CommunicateProviderImpl.java]: I don't like names with "Impl" part in it. Maybe better name would be FileCommunicateProvider

File: battleship-server/src/main/java/com/epam/solid/nie/config/language/LanguageConfigurationImpl.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.epam.solid.nie.config.language;
+
+import com.epam.solid.nie.config.Configuration;
+
+import java.util.ResourceBundle;
+
+public class LanguageConfigurationImpl implements LanguageConfiguration {
```
Comment (Ozymadias) [inline battleship-server/src/main/java/com/epam/solid/nie/config/language/LanguageConfigurationImpl.java]: Maybe use some "File" part in that name

Comment (rivalez) [reply battleship-server/src/main/java/com/epam/solid/nie/config/language/LanguageConfigurationImpl.java]: we will do this in the future commit

File: battleship-server/src/main/java/com/epam/solid/nie/config/language/CommunicateProviderImpl.java (hunk 1 above)
Comment (rivalez) [reply battleship-server/src/main/java/com/epam/solid/nie/config/language/CommunicateProviderImpl.java]: we will do this in the future commit

**Pull request merged**